llvm/lib/CodeGen/AsmPrinter/DwarfCompileUnit.h

```diff
@@ -151,7 +151,6 @@ class DwarfCompileUnit final : public DwarfUnit {
       DenseMap<DIGlobalVariable *, SmallVector<GlobalExpr, 1>> *p = nullptr) {
```

Comment:
Please also delete `globalVarMap` and `setGlobalVarMap`. They serve no purpose any more.

Reply:
Done.